solr/core/src/java/org/apache/solr/core/SolrConfig.java

```diff
@@ -810,6 +810,7 @@ public static class UpdateHandlerInfo implements MapSerializable {
     public final boolean openSearcher; // is opening a new searcher part of hard autocommit?
     public final boolean commitWithinSoftCommit;
     public final boolean aggregateNodeLevelMetricsEnabled;
+    public final String commitPollInterval;
```

Comment:
minor/subjective, to retain (partial) match order with constructor args
```suggestion
    public final String commitPollInterval;
    public final boolean aggregateNodeLevelMetricsEnabled;
```